inference/chat_completion.py

```diff
@@ -11,6 +11,7 @@
 
 from peft import PeftModel, PeftConfig
 from transformers import LlamaConfig, LlamaTokenizer, LlamaForCausalLM
+from optimum.bettertransformer import BetterTransformer
```

Comment:
Please remove from here as we have this in the try-catch-block for optional install

Reply:
Done